src/app-layer-ssh.h

```diff
@@ -28,5 +28,7 @@
 void RegisterSSHParsers(void);
 void SSHParserRegisterTests(void);
 
+int SSHTxLogCondition(ThreadVars *, const Packet *, void *state, void *tx, uint64_t tx_id);
```

Comment:
defined and not used in its commit

Reply:
Will move this code to logging commit.
Thanks.